lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -341,13 +341,36 @@ namespace Js
     bool
     FunctionBody::SkipAutoProfileForCoroutine() const
     {
-        return this->IsCoroutine() && CONFIG_ISENABLED(Js::JitES6GeneratorsFlag);
+#if defined(_ARM64_) || defined(_ARM_)
+        return false;
+#else
+        return this->IsCoroutine() && CONFIG_FLAG(JitES6Generators);
```

Comment:
instead of putting this behind architecture #ifdefs, you can change `DEFAULT_CONFIG_JitES6Generators` to be architecture specific

Reply:
Done